packages/react/src/components/Alert/Alert.tsx

```diff
@@ -13,7 +13,7 @@ export type AlertProps = {
    * Sets the size of the alert.
    * Does not affect font size.
    */
```

Comment:
Suggest we remove the jsdoc comments for all components using the default sizes so it uses the default jsdoc comment.

Reply:
We would have to do `.. & Size` for this to work:
![image](https://github.com/user-attachments/assets/f7ecd3ff-4256-497c-9f32-00d9054dedfc)

Reply:
Yeah, we talked about doing it like this in the previous issue we closed. This would have solved this for us. 

#2671

Reply:
`Size` currently does not have any description, do you have a suggestion for this?
I feel like it's better to have one for each component when we do it this way.

Reply:
We decided to drop the jsdoc for now, might add later.